### DISCUSSION TITLE
Adding more static

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1564] [Question and answer] [answered]
Is there any way to add a new static? I'm planning to add mortar for opfor.

Cheers.

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Yes https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L642-L643
Cheers
